src/eventlog/admin.py

```diff
@@ -1,17 +1,28 @@
 from django.contrib import admin
 from eventlog.models import LoginSession, Event
+from roster.models import Preference
+
 
 class SessionAdmin(admin.ModelAdmin):
     readonly_fields = ('id', 'user', 'startedAtTime', 'endedAtTime', 'userAgent')
     list_filter= ('user', 'startedAtTime')
     list_display = ('id', 'user', 'startedAtTime', 'endedAtTime')
     ordering = ('-startedAtTime',)
 
+
 class EventAdmin(admin.ModelAdmin):
     readonly_fields = ('id', 'eventTime', 'actor', 'group', 'membership', 'type', 'action', 'session',
                        'document', 'page', 'control', 'value')
     list_display = ('id', 'eventTime', 'actor', 'type', 'action')
     ordering = ('-eventTime',)
 
+
+class PreferenceAdmin(admin.ModelAdmin):
+    readonly_fields = ('id',)
+    list_display = ('id', 'user', 'pref', 'value')
+    ordering = ('user', 'pref')
```

Comment:
Shouldn't this be in roster/admin.py ?

Reply:
On the face of it, I agree - it looks like it's from your work to add the preference saving here: https://github.com/cast-org/clusive/blob/feature/CSB-520-save-preferences/src/eventlog/admin.py - did it need to be in here to administer the preference events?